# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/test/mochitest.ini
# Commit: f33e1ce6dcf1
# Full Hash: f33e1ce6dcf19e47952f3963c8e970c7c380acd2
# Author: Paul Adenot <paul@paul.cx>
# Date: 2023-07-20 21:19:23
# Regressor Bug: 1843003
# File Overlap Count: 2
# Description:
#   Bug 1843003 - Test that typical mp3 broadcast work well. r=alwu
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D183737
# ==============================================================================

diff -r 763b34de3c14 -r f33e1ce6dcf1 dom/media/test/mochitest.ini
--- a/dom/media/test/mochitest.ini	Thu Jul 20 13:17:58 2023 +0000
+++ b/dom/media/test/mochitest.ini	Thu Jul 20 13:17:59 2023 +0000
@@ -783,6 +783,10 @@
   hls/960x720_seg0.ts
   hls/960x720_seg1.ts
   sync.webm
+  two-xing-header-no-content-length.mp3
+  two-xing-header-no-content-length.mp3^headers^
+  single-xing-header-no-content-length.mp3
+  single-xing-header-no-content-length.mp3^headers^
 
 [test_capture_stream_av_sync.html]
 skip-if =
@@ -885,3 +889,4 @@
 tags = cloneelementvisually
 [test_cloneElementVisually_ended_video.html]
 tags = cloneelementvisually
+[test_mp3_broadcast.html]